browser/brave_content_browser_client.cc

```diff
@@ -1040,6 +1061,20 @@ BraveContentBrowserClient::CreateThrottlesForNavigation(
     throttles.push_back(std::move(debounce_throttle));
   }
 
+#if BUILDFLAG(ENABLE_REQUEST_OTR)
+  // Request Off-The-Record
+  if (auto request_otr_throttle =
+          request_otr::RequestOTRNavigationThrottle::MaybeCreateThrottleFor(
+              handle,
+              request_otr::RequestOTRServiceFactory::GetForBrowserContext(
+                  context),
+              EphemeralStorageServiceFactory::GetForContext(context),
+              user_prefs::UserPrefs::Get(context),
```

Comment:
this is ok, but in brave/browser it's preferable to use Profile::FromBrowserContext and then GetPrefs

Reply:
switched